scripts/detection/ssd/requirements.sh

```diff
@@ -0,0 +1 @@
+pip install pycocotools
```

Comment:
Can you please the NVIDIA version pycocotools, it is more optimized.

`pip --no-cache-dir --no-cache install 'git+https://github.com/NVIDIA/cocoapi#egg=pycocotools&subdirectory=PythonAPI'`

Reply:
good point, thanks!